scripts/update-copy-mod-doc-exceptions.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# This is a little wrapper around the python script
+# lint-copy-mod-doc.py
+
+find src archive -name '*.lean' | xargs ./scripts/lint-copy-mod-doc.py | sort > scripts/copy-mod-doc-exceptions.txt
```

Comment:
I appreciate the creative use of the `>` operator that makes `lint-copy-mod-doc.py` read its own output, which happens to be empty at first.

Reply:
Is it too evil?

Reply:
It should be be fine.  I don't know if the shell guarantees to open the file before launching the subprocesses, but even if it doesn't `find` takes long enough to prevent the race condition in practice.

Reply:
Hmm, it looks like the `>` line got removed. I'd prefer to have that done in this script. I'll open a new PR.